docs/tutorials/using-host-labels.md

```diff
@@ -0,0 +1,219 @@
+		"_kernel_version": "4.19.0-6-amd64",
+		"_os_version": "10 (buster)",
+		"_os_name": "Debian GNU/Linux",
+		"type": "webserver",
+    "location": "seattle",
+    "provider": "aws"
+	},
+  ...
+}
+```
+
+You may have noticed a handful of labels that begin with an underscore (`_`). These are automatic labels.
+
+### Automatic labels
+
+When Netdata starts, it captures relevant information about the system and converts them into automatically-generated
+host labels. You can these use these to logically organize your systems via health entities, exporting metrics,
+streaming/master status, and more.
+
+They capture the following:
+
+-   Kernel version
+-   Operating system name and version
+-   CPU architecture, system cores, CPU frequency, RAM, and disk space
+-   If Netdata is running inside of a container
+    -   If so, OS and hardware details about the container's host
+-   What virtualization layer the system runs on top of, if any
+-   Whether the system is a streaming master or slave
+
+If you want to organize your systems without manually creating host tags, try the automatic labels in some of the
+features below.
+
+## Host labels in streaming
+
+You may have noticed the `_is_master` and `_is_slave` automatic labels from above. Host labels are also now streamed
+from a slave to its master agent, which concentrates an entire infrastructure's OS, hardware, container, and
+virtualization information in one place: the master.
+
+Now, if you'd like to remind yourself of how much RAM a certain slave system has, you can simply access
+`http://localhost:1999/host/SLAVE_NAME/api/v1/info` and reference the automatically-generated host labels from the slave
+system. It's a vastly simplified way of accessing critical information about your infrastructure.
+
+You can also use `_is_master`, `_is_slave`, and any other host labels in both health entities and metrics exporting.
+Speaking of which...
+
+## Host labels in health entities
+
+You can use host labels to logically organize your systems by their type, purpose, or location, and then apply specific
+alarms to them.
+
+For example, let's use configuration example from earlier:
+
+```conf
+[host labels]
+    type = webserver
+    location = us-seattle
+    provider = aws
+```
+
+You could now create a new health entity (checking if disk space will run out soon) that applies only to any host
+labeled `webserver`:
+
+```yaml
+    template: disk_fill_rate
+          on: disk.space
+      lookup: max -1s at -30m unaligned of avail
+        calc: ($this - $avail) / (30 * 60)
+       every: 15s
+ host labels: type = webserver
+```
+
+Or, by using one of the automatic labels, for only webserver systems running a specific OS and version:
+
+```yaml
+    template: disk_fill_rate
+          on: disk.space
+      lookup: max -1s at -30m unaligned of avail
+        calc: ($this - $avail) / (30 * 60)
+       every: 15s
+ host labels: type = webserver, _os_version = 1*, _os_name = Debian*
```

Comment:
The `host labels` option uses simple patterns, but the simple patterns don't allow commas. @thiagoftsm am I wrong?

Reply:
> Simple patterns are a space separated list of words,

Reply:
I see now from #7600 that there is no support for commas or `AND` to use multiple host labels. I think I used the commas as an artifact of seeing the `AND` in that PR when Thiago was working on it.

I'll change these examples to simpler ones.

Reply:
You are correct @vlvkobal !